ote_sdk/ote_sdk/test_suite/training_tests_actions.py

```diff
@@ -310,6 +309,16 @@ def create_openvino_task(model_template, environment):
     return openvino_task
 
 
+def get_task_class(path):
```

Comment:
copy (not move!) https://github.com/openvinotoolkit/training_extensions/blob/develop/ote_cli/ote_cli/utils/importing.py to ote_sdk/ote_sdk/ and use function from there.

Reply:
Done!